app/build.gradle

```diff
@@ -77,32 +77,41 @@ dependencies {
      * Import the PingOne for Customers Android SDK. Get the latest version from
      * https://search.maven.org/search?q=g:com.pingidentity.pingonemfa
      */
-    implementation 'com.pingidentity.pingonemfa:android-sdk:1.10.0'
+    implementation 'com.pingidentity.pingonemfa:android-sdk:1.11.0-SNAPSHOT'
```

Comment:
Should NOT be snapshot, this is the release version

Reply:
Done